planetiler-basemap/src/main/java/com/onthegomap/planetiler/basemap/BasemapProfile.java

```diff
@@ -104,37 +120,53 @@ public BasemapProfile(Translations translations, PlanetilerConfig config, Stats
       }
     }
 
-    // pre-process layers to build efficient indexes for matching OSM elements based on matching expressions
+    // pre-process layers to build efficient indexes for matching OSM elements based on matching
+    // expressions
     // Map from imposm3 table row class to the layers that implement its handler.
     var handlerMap = Tables.generateDispatchMap(layers);
-    osmMappings = Tables.MAPPINGS
-      .mapResults(constructor -> {
-        var handlers = handlerMap.getOrDefault(constructor.rowClass(), List.of()).stream()
-          .map(r -> {
-            @SuppressWarnings("unchecked") var handler = (Tables.RowHandler<Tables.Row>) r.handler();
-            return handler;
-          })
-          .toList();
-        return new RowDispatch(constructor.create(), handlers);
-      }).simplify().index();
-    wikidataMappings = Tables.MAPPINGS
-      .mapResults(constructor ->
-        handlerMap.getOrDefault(constructor.rowClass(), List.of()).stream()
-          .anyMatch(handler -> !IgnoreWikidata.class.isAssignableFrom(handler.handlerClass()))
-      ).filterResults(b -> b).simplify().index();
+    osmMappings =
+        Tables.MAPPINGS
+            .mapResults(
+                constructor -> {
+                  var handlers =
+                      handlerMap.getOrDefault(constructor.rowClass(), List.of()).stream()
```

Comment:
😢  I wonder if this insane formatting here is to discourage you from writing code like this...

Reply:
The annotation in the middle of it all is a nice touch 😂

Reply:
Besides, it has a cyclomatic complexity of 1, who could complain?  😂